packages/odyssey-react/src/components/Button/Button.module.scss

```diff
@@ -0,0 +1,211 @@
+/*!
+ * Copyright (c) 2021-present, Okta, Inc. and/or its affiliates. All rights reserved.
+ * The Okta software accompanied by this notice is provided pursuant to the Apache License, Version 2.0 (the "License.")
+ *
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0.
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *
+ * See the License for the specific language governing permissions and limitations under the License.
+ */
+
+.root {
+  @include border-radius($base-border-radius);
+
+  display: inline-block;
+  position: relative;
+  margin-block: 0;
+  margin-inline: 0;
+  padding-block: $spacing-xs-em;
+  padding-inline: $spacing-s-em;
+  transition-property: color, background-color, border-color, box-shadow;
+  transition-duration: 100ms;
+  transition-timing-function: linear;
+  border: 1px solid transparent;
+  background-color: $color-primary-base;
+  box-shadow: 0 0 0 0 $color-primary-light;
+  color: $white;
+  font-family: $body-font-family;
+  font-size: ms(0);
+  font-weight: 600;
+  line-height: $base-line-height;
+  white-space: nowrap;
+
+  &:hover,
+  &.stateHover {
+    border-color: $color-primary-dark;
+    background-color: $color-primary-dark;
+  }
+
+  &:focus,
+  &.stateFocus {
+    @include outline;
+
+    background-color: $color-primary-dark;
+  }
+
+  &:disabled {
+    border-color: transparent;
+    background-color: $color-primary-light;
+    cursor: not-allowed;
+  }
+
+  & + & {
+    margin-inline-start: $spacing-s;
+  }
+}
+
+// Parts
+
+.label {
+  display: inline-block;
+  min-width: $spacing-l-em;
+
+  &:not(:only-child) {
+    margin-inline-start: $spacing-xs;
+  }
+}
+
+// Variants
+
+.variantSecondary {
```

Comment:
As Button has alternative styles based on independent dimensions, I've prefixed those additive styles for clarity. Currently, I've opted to maintain the additive style instead of trying to compose classes for all possible variants. This should help keep our bundle size smaller and (hopefully) increase clarity.